.gitignore

```diff
@@ -0,0 +1,2 @@
+Gemfile.lock
```

Comment:
We should leave the Gemfile.lock under git. Having the Gemfile.lock used few years ago to be the norm but the community shitfted to include it in libraries as it make things easier to contribute to a project, bisect it etc ... https://bundler.io/blog/2018/01/17/making-gem-development-a-little-better.html